.github/workflows/continuous-integration.yml

```diff
@@ -45,3 +45,46 @@ jobs:
         run: yarn --immutable
       - name: Build Code
         run: yarn build
+
+  testing:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+      - name: Use Node.js v16
+        uses: actions/setup-node@17f8bd926464a1afa4c6a11669539e9c1ba77048 # tag=v3
+        with:
+          node-version: 16
+          cache: yarn
+          registry-url: https://registry.npmjs.org/
```

Comment:
I have not seen Yarn used instead of NPM anywhere, why?

Reply:
Consistency honestly. I've been changing it here and there in favware/sapphire/skyra repos ever since changing from `npm publish` to `yarn npm publish` also because of consistency. Also for publishing there is the very slight quirk that if you publish with `yarn npm publish` or set the yarnpkg registry it shows up sooner on that website and personally I find the yarn website waaaaaaaaay better to navigate because they use Algolia instant search instead of the slow and inaccurate search that npm uses.